pilot/pkg/xds/debug.go

```diff
@@ -698,6 +704,9 @@ func (s *DiscoveryServer) connectionConfigDump(conn *Connection, includeEds bool
 		v3.SecretType,
 		v3.EndpointType,
 		v3.ExtensionConfigurationType,
+		v3.WorkloadType,
+		v3.AddressType,
+		v3.WorkloadAuthorizationType,
```

Comment:
why do we need to change the default? It is meant to mirror envoy's

Reply:
I understand `no types parameter` mean dump all configurations.

Reply:
I don't think so, its already
```
	// EDS is disabled by default for compatibility with Envoy config_dump interface
	if includeEds {
```

for example. This interface is used for `istioctl ps` to diff envoy vs pilot

Reply:
make sense, reverted.